### PR TITLE
Fix various javadoc warnings

### PR DESCRIPTION
**Pull request by cbornet**

### CODE REVIEW

File: pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/EndOfStreamAction.java [hunk 1]
```diff
@@ -16,6 +16,11 @@
 
 package org.apache.pulsar.reactive.client.api;
 
+/**
+ * Action to perform once the end of the stream is reached.
+ *
+ * @author Lari Hotari
```
Comment (tisonkun) [inline pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/EndOfStreamAction.java:22]: IIRC generally, we don't encourage using the `@author` tag. Is there any reason for doing this?

Comment (cbornet) [reply pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/EndOfStreamAction.java:22]: It's in the checkstyle rules (derivative of Spring rules)
If we want to remove it, we should do it in another PR.

Comment (lhotari) [reply pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/EndOfStreamAction.java:22]: I filed #67 to remove author tags. I think it's better to get rid of them.

**Review by lhotari — APPROVED** [review]

File: pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/EndOfStreamAction.java (hunk 1 above)
Comment (lhotari) [reply pulsar-client-reactive-api/src/main/java/org/apache/pulsar/reactive/client/api/EndOfStreamAction.java:22]: Let's get the in-process PRs merged and then handle that.

**Pull request merged**